lib/protocol/networks.js

```diff
@@ -775,6 +806,11 @@ regtest.block = {
   slowHeight: 0
 };
 
+regtest.airdrop = {
+  disableGoosig: false,
+  disableGoosigHeight: 0
```

Comment:
Maybe we should make this default to `-1`. The codebase typicallys uses `-1` for default, unset heights. Also, a height of `0` implies goosig is always disabled. I understand the boolean parameter should protect against this false interpretation of the height, but it seems safer to set it to `-1`.

Reply:
Updated to set it as `-1`. If we want to get rid of the boolean and just have `-1` mean never disable, we could do that as well. I wanted to be explicit when writing this.